### PR TITLE
Babel 7.14

### PR DESCRIPTION
**Pull request by JLHwung**
Updated `@babel/parser` to 7.14
Added new plugins supported in `@babel/parser`.

### CODE REVIEW

File: website/src/parsers/js/babylon7.js [hunk 1]
```diff
@@ -45,9 +42,7 @@ export const defaultOptions = {
   ranges: false,
   tokens: false,
   plugins: [
-    'classProperties',
-    'classPrivateProperties',
-    'classPrivateMethods',
+    'classStaticBlock',
```
Comment (JLHwung) [inline website/src/parsers/js/babylon7.js:45]: Shipped in V8.

File: website/src/parsers/js/babylon7.js [hunk 2]
```diff
@@ -16,9 +16,7 @@ const availablePlugins = [
 
   // ECMAScript Proposals
   // https://babeljs.io/docs/en/babel-parser.html#ecmascript-proposalshttpsgithubcombabelproposals
-  'classProperties',
-  'classPrivateProperties',
-  'classPrivateMethods',
```
Comment (JLHwung) [inline website/src/parsers/js/babylon7.js:21]: They are materialized as part of ES2022.

File: website/src/parsers/js/babylon7.js [hunk 3]
```diff
@@ -27,7 +25,6 @@ const availablePlugins = [
   'functionBind',
   'importAssertions',
   'moduleBlocks',
-  'moduleStringNames',
```
Comment (JLHwung) [inline website/src/parsers/js/babylon7.js:30]: 'moduleStringNames' is materialized.

File: website/src/parsers/js/babylon7.js [hunk 4]
```diff
@@ -99,7 +94,6 @@ export default {
   parse(babylon, code, options) {
     options = {...options};
     // TODO: Make decoratorsBeforeExport settable through settings somehow
-    // TODO: Make pipelineOperator.proposal settable through settings somehow
```
Comment (JLHwung) [inline website/src/parsers/js/babylon7.js:102]: This TODO item is done.

**Comment by fkling** [comment]
Thank you. Sorry that I forgot about this.

**Pull request merged**